doc/setting_up_sidewalk_environment/setting_up_sdk.rst

```diff
@@ -42,6 +24,8 @@ Sidewalk as an nRF Connect SDK repository
 
 Follow these steps to download the Sidewalk application for the nRF Connect SDK:
 
+.. Instructions tbc
+
```

Comment:
Could you please verify the below instructions? I think we will need to modify it since it starts with the usage of TCM

Reply:
I think we need to remove the step 1. with  ` .. figure:: /images/toolchain_manager.png` as the tool is deprecated now

Reply:
Removed from both regular and manifest repository instructions, please double-check